RtAudio.cpp

```diff
@@ -82,15 +82,12 @@ const unsigned int RtApi::SAMPLE_RATES[] = {
     return s;
   }
 
-#elif defined(__LINUX_ALSA__) || defined(__LINUX_PULSE__) || defined(__UNIX_JACK__) || defined(__LINUX_OSS__) || defined(__MACOSX_CORE__)
+#elif defined(__unix__) || defined(__APPLE__)
```

Comment:
shouldn't this just be `#else` ?
There are non-unix POSIX systems that have pthreads support (HaikuOS is an example) and there is no fallback `#else` here anymore..

Reply:
Fair point. This should be changed for #308 too.

Reply:
I generally agree with this, I don't think trying to enumerate any and all systems that support pthreads is necessarily the right approach and since there is no other fall-back, I guess simply `#else` could work.   Maybe in the future we can switch to C++11 threads and not worry about this kind of thing?  Care to make a PR @falkTX?

Reply:
No worries, done.

Reply:
I think just changing it to always fallback to pthread is fine tbh, rtaudio would not function without it anyway.
Just need to be mentioned somewhere that this is a requirement.